swift/integration-tests/frontend-invocations/Makefile

```diff
@@ -0,0 +1,10 @@
+# TODO: Add linux
+SDK=$(shell xcrun -show-sdk-path)
+FRONTEND=$(shell xcrun -find swift-frontend)
+
+all:
+	$(FRONTEND) -frontend -c A.swift -sdk $(SDK)
+	$(FRONTEND) -frontend -c B.swift -o B.o -sdk $(SDK)
+	$(FRONTEND) -frontend -c -primary-file C.swift -sdk $(SDK)
+	$(FRONTEND) -frontend -c -primary-file D.swift -o D.o -sdk $(SDK)
+	$(FRONTEND) -frontend -c -primary-file E.swift Esup.swift -o E.o -sdk $(SDK)
```

Comment:
is the makefile necessary? Couldn't we simply move this list of commands to the `test.py` script?

Reply:
It's just easier with a Makefile/bash in this specific case (due to `swift-frontend`/`-sdk` resolution).